examples/multi-ctx/main.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"runtime"
+	"strings"
+	"sync"
+	"time"
+
+	_ "github.com/go-python/gpython/modules"
```

Comment:
as this is an example, perhaps comment on why this is needed?

Reply:
great suggestion